src/client.js

```diff
@@ -60,3 +61,27 @@ function refresh_token (_refresh_token) {
             return result;
         });
 }
+
+function revoke_token (token, client_id, type = '') {
+    const scopes_string = cache.find_key('client', {token});
+    const payload = {
+        client_id: client_id,
+        type: type,
+    };
+    const access_token = `Bearer ${token}`;
+
+    return cudl.post
+        .to(config.base_url + config.path + '/oauth/revoke')
+        .set_header('Authorization', access_token)
+        .send(payload)
+        .max_retry(config.retry_count)
+        .promise()
+        .then(result => {
+            if (scopes_string) {
+                cache.forget('client', token);
```

Comment:
Does this work? Looks like the cache is cached with the key of "scopes_string" so this should be 
```
cache.forget('client', scopes_string)
```
instead? (or perhaps rename the scopes_string to key?

Reply:
Sure. I'll rename scopes to key and use it instead.

Reply:
This is now already updated. :smile: